First/src/MediumDifficultyAssignmnets/AnnualSalary1.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Scanner;
+
+public class AnnualSalary1 {
+    public static void main(String[] args) {
+        float income;
+        float tax;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter your income ");
+
+        income = sc.nextFloat();
+        if (income <= 500000) {
+            System.out.println("You are Tax free");      //Nill
+        } else if (income >= 500000 && income < 700000) //both true then if
+        {
+            float amount = income - 500000;     //declare variable    6L-5L=1L
```

Comment:
Use just one global variable - amount

Reply:
Please check and change the logic for tax calculation. If possible paste output with all possible values.

Reply:
Done !